index.html

```diff
@@ -805,10 +807,13 @@ <h2>The Context</h2>
     terms to communicate with one another more efficiently, but without
     losing accuracy.</p>
 
-    <p>Simply speaking, a <a>context</a> is used to map <a>terms</a> to
-      <a>IRIs</a>. <a>Terms</a> are case sensitive
-      and any valid <a>string</a> that is not a reserved JSON-LD <a>keyword</a>
-      can be used as a <a>term</a>.</p>
+    <p>Simply speaking, a <a>context</a> is used to map <a>terms</a> to <a>IRIs</a>.
+      <a>Terms</a> are case sensitive and most any valid <a>string</a> that is not a reserved JSON-LD <a>keyword</a>
+      can be used as a <a>term</a>.
+      <span class="changed">The empty string `""` and strings that have the form
+        of a keyword (e.g., staring with `"@"`) must not be used as terms.
+        Strings that have the form of
+        an absolute IRI (e.g., containing a `":"`) should not be used as terms.</span></p>
```

Comment:
> Strings that [contain a colon] should not be used as terms.

Well, it is acceptable to have a *term definition* of the form `"foaf:knows": {"@type": "@id"}`.
Isn't `foaf:knows` a term in that case?

Reply:
My bad, the key for a term definition is not necessarily a term. Slightly confusing, but ok :)